android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -232,7 +207,18 @@ private void generateLocalBroadcast(JSONObject referringParams,
             }
         }.init(reactActivity);
 
-        branch.initSession(referralInitListener, uri, reactActivity);
+        notifyJSOfInitSessionStart(reactActivity, uri);
+        branch.sessionBuilder(reactActivity).withCallback(referralInitListener).withData(uri).init();
```

Comment:
I believe `sessionBuilder` is a static method, so shouldn't be called on an instance object

Reply:
:+1: Thanks. This seems to work in testing. I think C++ and Java are not as fussy as other languages about static/member calling. That is, a static member is always available to instances. Certainly true in C++. In Obj-C, e.g. this sort of error would fail to compile. I'll fix it.